README.md

```diff
@@ -55,6 +55,8 @@ If you need to use the Zendesk service you need to add 3 environment variables
 
 Add any registers using the `/admin` UI.
 
+If you need to redownload an existing register from scratch, you can run the job `bundle exec rake populate_db:force_full_register_download[SLUG]` where `SLUG` is the slug of the register.
```

Comment:
I think we are missing `registers_frontend:` off the front of the rake task?

Reply:
Doh. Added this in